src/status_im2/contexts/onboarding/common/overlay/view.cljs

```diff
@@ -45,9 +45,12 @@
         max-blur-amount
         2))))
 
+;; we had to register it here, because of hotreload, overwise on hotreload it will be reseted
+(defonce blur-amount (reagent/atom 0))
```

Comment:
is it possible to mark this private so it doesn't get used outside this file at least? 🤔